content/tracking/tracking_1.md

```diff
@@ -165,6 +165,46 @@ export class SnowplowService {
   })
 }
 ```
+{{% /tab %}}
+{{% tab name="GTM" %}}
+❗❗**NOTE: A Google Tag Manager Account is required for this pathway**
+
+#### **Step 1:** Create container
+Create a web container for your site.
+![Create container](../images/create_container.png?classes=shadow)
+
+#### **Step 2:** Add Tags to site
+Install Google Tag Manager on your site by following the [installation guide](https://tagmanager.google.com/) as below:
+
+![Install GTM](../images/google_tag.png)
+
+#### **Step 3:** Add New Template Variable
+Go to `Templates` and click `Search Gallery` under `Variable Templates`.
+![Variable Template](../images/add_variable_template.png?classes=shadow)
+
+Search `Snowplow V3 Settings` in the search bar.
+![Variable Search](../images/variable_search.png?classes=shadow)
+
+Add the template to your workspace.
+![Add to workspace](../images/template_to_workspace.png?classes=shadow)
+
+Navigate to Variables, and edit the `Snowplow v3 Settings`
+![Template Variables](../images/user_defined_variables.png?classes=shadow)
```

Comment:
I don't think you'll just have an existing one after you've added the template to your workspace. You'll need to make a new one rather than editing an existing one.

Reply:
Good point, I've changed the screenshot so that it's clearer.